app/scripts/migrations/092.2.test.ts

```diff
@@ -0,0 +1,99 @@
+import { NetworkType } from '@metamask/controller-utils';
+import { NetworkStatus } from '@metamask/network-controller';
+import { migrate, version as currentStateVersion } from './092.2';
+
+const TEST_NETWORK_CONTROLLER_STATE = {
+  selectedNetworkClientId: 'test-network-client-id',
+  networkId: 'sdf',
+  providerConfig: {
+    type: NetworkType.rpc,
+    chainId: '0x9393',
+    nickname: 'Funky Town Chain',
+    ticker: 'ETH',
+    id: 'test-network-client-id',
+  },
+  networkConfigurations: {
+    'network-configuration-id-1': {
+      chainId: '0x539',
+      nickname: 'Localhost 8545',
+      rpcPrefs: {},
+      rpcUrl: 'http://localhost:8545',
+      ticker: 'ETH',
+    },
+  },
+  networksMetadata: {
+    'test-network-client-id': {
+      status: NetworkStatus.Available,
+      EIPS: {},
+    },
+  },
+};
+
+const anyPreviousStateVersion = Math.floor(
```

Comment:
Nit: Perhaps we could use a static version here instead? I can see how a random version might cover a wider range, but it would be frustrating if this did cause a failure because the error message likely wouldn't include the chosen version anyway.